controllers/external.go

```diff
@@ -0,0 +1,59 @@
+package controllers
+
+import (
+	"challenge/api/errorsHandlers"
```

Comment:
By convention, the packages should be ordered as follows:
Go core

Project Local 

Third-Party 

Do this in all the files.